ktor-server/ktor-server-core/jvm/src/io/ktor/server/http/content/StaticContent.kt

```diff
@@ -72,22 +73,32 @@ public class StaticContentConfig<Resource : Any> internal constructor() {
 
     /**
      * Configures [ContentType] for requested static content.
-     * For files, [Resource] is a request [File].
-     * For resources, [Resource] is a path to requested resource.
+     * For files, [Resource] is a requested [File].
+     * For resources, [Resource] is a [URL] to requested resource.
      */
     public fun contentType(block: (Resource) -> ContentType) {
         contentType = block
     }
 
     /**
      * Configures [CacheControl] for requested static content.
-     * For files, [Resource] is a request [File].
-     * For resources, [Resource] is a path to requested resource.
+     * For files, [Resource] is a requested [File].
+     * For resources, [Resource] is a [URL] to requested resource.
      */
     public fun cacheControl(block: (Resource) -> List<CacheControl>) {
         cacheControl = block
     }
 
+    /**
+     * Configures modification of a call for requested content.
+     * Useful to add headers to the response, such as [HttpHeaders.ETag]
+     * For files, [Resource] is a requested [File].
+     * For resources, [Resource] is a [URL] to requested resource.
+     */
```

Comment:
Shouldn't there be articles here?
Configures modification of a call for **the** requested content.
For resources, [Resource] is a [URL] to **the** requested resource.